sources/packages/web/src/components/students/applicationTracker/MultipleDisbursementBanner.vue

```diff
@@ -156,7 +156,7 @@ export default defineComponent({
     },
     coeDenialReason: {
       type: String,
-      required: true,
+      required: false,
```

Comment:
why did we change this?

Reply:
Volar extension get the issue where it was required but it was having a nullable value assigned.